about/getting_started.markdown

```diff
@@ -60,6 +60,12 @@ Valid keys are as follows:
     <tr><td>dnsmadeeasy_secret_key</td><td></td></tr>
     <tr><td>go_grid_api_key</td><td></td></tr>
     <tr><td>go_grid_shared_secret</td><td></td></tr>
+    <tr><td>google_project</td><td></td></tr>
+    <tr><td>google_client_email</td><td></td></tr>
+    <tr><td>google_key_location</td><td></td></tr>
+    <tr><td>google_key_string</td><td></td></tr>
+    <tr><td>google_json_key_location</td><td></td></tr>
+    <tr><td>google_json_key_string</td><td></td></tr>
```

Comment:
Description: In-line JSON private key string contents (alternative to specifying path)